app/assets/stylesheets/base.scss

```diff
@@ -15,6 +15,15 @@ a {
   }
 }
 
+:disabled {
+  // scss-lint:disable ImportantRule
+  background-color: $sul-btn-disabled-color !important;
```

Comment:
There is an `a.disabled` styling above, but I don't see anywhere we have links w/ a disabled class on it (and I think its intention is different than the disabled button (e.g color is disabled instead of background).